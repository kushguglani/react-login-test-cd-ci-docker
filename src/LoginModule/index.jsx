import React from 'react'
import { UserLogin } from './hooks/userLogin'
import Login from './component/Login'

export default function LoginModule() {
    const { state, submitLogin } = UserLogin();
    return (
       <Login state={state} submitLogin={submitLogin} />
    )
}
