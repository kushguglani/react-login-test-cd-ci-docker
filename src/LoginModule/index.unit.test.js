import React from 'react';
import { render } from '@testing-library/react'
import LoginModule from './';

describe('Login Module', () => {
    it('matches snapshot', () => {
        const { container } = render(<LoginModule />);
        expect(container.cloneNode(true)).toMatchSnapshot();
    })
})