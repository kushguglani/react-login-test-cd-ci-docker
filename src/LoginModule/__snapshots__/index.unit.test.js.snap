// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login Module matches snapshot 1`] = `
<div>
  <fieldset>
    <legend>
      Login Form
    </legend>
    User Name : 
    <input
      name="userName"
      placeholder="User Name"
      type="text"
    />
    <br />
    Password : 
    <input
      name="password"
      placeholder="Password"
      type="password"
    />
    <br />
    <br />
    <button>
      Login
    </button>
  </fieldset>
</div>
`;
